src/multiselect/multiselect-internal.tsx

```diff
@@ -60,6 +60,8 @@ type MultiselectBaseProps = {
   inverted?: boolean;
   /* Handler called when a dropdown Item is selected, or an active Tag is dismissed. Use to update the `isActive` prop for Items in `options`. */
   onChange: (id: string) => void;
+  /* Handler called a user types into the input of the Multiselect. Can be used to apply custom filtering behavior. */
```

Comment:
If you use [JSDoc syntax](https://jsdoc.app/about-getting-started.html), VSCode IntelliSense will pick it up and show it in a tooltip.

```suggestion
  /** Handler called a user types into the input of the Multiselect. Can be used to apply custom filtering behavior. */
```